packages/styles/package.json

```diff
@@ -16,9 +16,16 @@
     "components",
     "react"
   ],
+  "files": [
+    "scss",
+    "index.scss"
+  ],
   "publishConfig": {
     "access": "public"
   },
+  "peerDependencies": {
+    "sass": "^1.32.12"
+  },
```

Comment:
Based on https://github.com/carbon-design-system/carbon/issues/10600 we had said we'd be requiring `sass@>=1.33.0`